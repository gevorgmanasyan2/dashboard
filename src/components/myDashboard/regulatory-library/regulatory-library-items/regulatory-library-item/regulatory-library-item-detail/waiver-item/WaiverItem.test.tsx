/* eslint-disable @typescript-eslint/no-empty-function */
/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import WaiverItem from './WaiverItem'

it('should render WaiverItem component', () => {
  const mockItem = {
    name: 'zxcv',
    description: 'zxcv',
    imageSource: '/waiver_librery-item/5/display/',
    actionLink: '/waiver_librery-item/5/download/',
    fileName: '',
    externalLink: '',
  }
  expect(shallow(<WaiverItem {...mockItem} />)).toMatchSnapshot()
})
