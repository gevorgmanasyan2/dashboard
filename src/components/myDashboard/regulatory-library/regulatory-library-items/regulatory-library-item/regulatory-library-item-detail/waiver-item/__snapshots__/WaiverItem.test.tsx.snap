// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render WaiverItem component 1`] = `
<div
  className="waiver__item__container__modal__details"
>
  <div>
    <img
      alt=""
      src=""
      style={
        Object {
          "height": "200px",
          "width": "399px",
        }
      }
    />
  </div>
  <div
    className="waiver__item__container__modal__details__file"
  >
    <ForwardRef(FileOutlined) />
    <span
      className="waiver__item__container__modal__details__file--name"
    >
      zxcv
    </span>
  </div>
  <div
    className="waiver__item__container__modal__details--desc"
  >
    <p>
      zxcv
    </p>
  </div>
  <div
    className="waiver__item__container__modal__details__footer"
  >
    <div
      className="waiver__item__container__modal__details__footer--icon"
    >
      <Button
        block={false}
        ghost={false}
        href=""
        htmlType="button"
        loading={false}
        target="_blank"
      >
        <ForwardRef(BookOutlined) />
      </Button>
    </div>
  </div>
</div>
`;
